### DISCUSSION TITLE
JSON schema of SysML model in database

### DISCUSSION THREAD

**Discussion by Krusty84** [eclipse-syson/syson#328] [Q&A] [answered]
Hi Dear Team,
The SysML v2 Models storing in database in JSON format:

`
{
  "json": {
    "version": "1.0",
    "encoding": "utf-8"
  },
  "ns": {
    "sysml": "http://www.eclipse.org/syson/sysml"
  },
  "content": [
    {
      "id": "94e0771e-8c48-470c-b2b5-f96e733edcd4",
      "eClass": "sysml:Package",
      "data": {
        "declaredName": "SimpleVehicleModel",
        "elementId": "0707780d-3a6d-40af-a08e-27ade492fd4d",
        "ownedRelationship": [
          {
            "id": "b8b258ed-483a-4861-972b-8ccf704a71b6",
            "eClass": "sysml:NamespaceImport",
            "data": {
              "declaredName": "Definitions",
              "elementId": "36262f05-22cb-4dbb-b821-23b8c02b2023",
              "target": [
                "4e5db5f4-ab59-481a-a45a-bc6d272e3cea"
              ],
              "importedNamespace": "4e5db5f4-ab59-481a-a45a-bc6d272e3cea"
            }
          },
          {
            "id": "3eda9130-ddf2-4b8b-ac87-b2a0fa668a2f",
            "eClass": "sysml:NamespaceImport",
....
`

Where can get JSON schema for that?

Thank you/Merci!

**Comment by AxelRICHARD** [comment, marked as answer]
Hello,
There is no JSON schema for that (in the way you can imagine), we just rely on https://github.com/eclipse-sirius/sirius-emf-json to store our ecore-based models in json.
Regards,
